### DISCUSSION TITLE
NEW we-act mini stm32h723vgt6 board variant

### DISCUSSION THREAD

**Discussion by admech1** [stm32duino/Arduino_Core_STM32#2207] [Variants] [answered]
Hi i am looking for direction or a good tutorial on how to add this board to the arduino boards list.
There are 2 other variants already supported of the we act mini,  they all contain the same hardware except the mcu
any help would be appreciated

**Comment by fpistm** [comment, marked as answer]
Hi @admech1 
Some info here:
https://github.com/stm32duino/Arduino_Core_STM32/wiki/Add-a-new-variant-%28board%29

And here the PR you can used as examples:
https://github.com/stm32duino/Arduino_Core_STM32/pull/1552
